src/ios/ZeroConf.swift

```diff
@@ -436,4 +433,36 @@ import Foundation
         return nil
     }
 
+    fileprivate static func dictionary(fromTXTRecord txtData: Data) -> [String: String] {
+
+        var result = [String: String]()
+        var data = txtData
+
+        while !data.isEmpty {
```

Comment:
Is it possible to get stuck inside this `while`? If there is a timeout, how long does it take? We have to be aware of (and prevent if possible) long-running Cordova requests.